luigi/worker.py

```diff
@@ -727,11 +730,19 @@ def update_tracking_url(tracking_url):
                 tracking_url=tracking_url,
             )
 
+        def update_status_message(message):
+            self._scheduler.set_task_status_message(task.task_id, message)
+
+        run_callbacks = {
+            "tracking_url_callback": update_tracking_url,
+            "status_message_callback": update_status_message
+        }
```

Comment:
I imagined this being a `__call__`able object where it by default does "update_tracking_url" and prints a deprecation warning.